crates/re_types_builder/src/codegen/python/space_views.rs

```diff
@@ -0,0 +1,174 @@
+use crate::{
+    codegen::{
+        common::StringExt as _,
+        python::{quote_doc_lines, quote_obj_docs},
+    },
+    Object, Objects, Reporter, ATTR_PYTHON_ALIASES, ATTR_RERUN_SPACE_VIEW_IDENTIFIER,
+};
+
+pub fn code_for_space_view(reporter: &Reporter, objects: &Objects, obj: &Object) -> String {
+    assert!(obj.is_struct());
+
+    let mut code = String::new();
+
+    code.push_indented(
+        0,
+        "
+from .. import archetypes as blueprint_archetypes
```

Comment:
this assumes space views are always in the blueprint scope. Probably a safe assumption but probably worth adding the assert somewhere.

Reply:
somewhat circular: we identify something bein `ObjectKind::View` if `pkg_name.starts_with("rerun.blueprint.views")` is true